### DISCUSSION TITLE
Windows firewall detects Win32/Wacapew.C!ml trojan

### DISCUSSION THREAD

**Discussion by sinaasadiyan** [hiddify/hiddify-app#567] [Q&A]
Windows firewall detects Win32/Wacapew.C!ml trojan while installing hiddify 16.7. Is this harmful? any solution for that.
I am going back to 16< versions

**Comment by smaga38** [comment]
[VirusTotal](https://www.virustotal.com/gui/file/4c0fba7c6f8cea429423787b25ab1133f84182f5f7a84ce0fe4278ac8e454d4a?nocache=1) detects the installer as Win32.Trojan.Agent.0WVQD3 and Trojan.WinGo.Rozena

Why hasn't a single request regarding this or similar topics received a response from the developers?
